app/assets/javascripts/admin/order_cycles/directives/change-warning.js.coffee

```diff
@@ -0,0 +1,19 @@
+angular.module("admin.orderCycles").directive "changeWarning", (ConfirmDialog) ->
+  restrict: "A"
+  scope:
+    orderCycle: '=changeWarning'
+  link: (scope, element, attrs) ->
+    acknowledged = false
+    cancel = 'admin.order_cycles.date_warning.cancel'
+    proceed = 'admin.order_cycles.date_warning.proceed'
+    msg = 'admin.order_cycles.date_warning.msg'
+    options = { cancel: t(cancel), confirm: t(proceed) }
+
+    element.focus ->
+      count = scope.orderCycle.subscriptions_count
+      return if acknowledged
+      return if moment(scope.orderCycle.orders_close_at).isBefore()
+      return if count < 1
```

Comment:
doesn't it make sense to collapse these conditionals into a single if statement?

Reply:
Then you get:
```ruby
return if acknowledged ||
         moment(scope.orderCycle.orders_close_at).isBefore() ||
         count < 1
```
I actually find Rob's version easier to read, because you can look at each line separately without thinking about logic operators. :man_shrugging: Both valid, I would say.